src/features/usage-consolidation-accordion/summary-table/column-properties.js

```diff
@@ -21,11 +27,20 @@ const DEFAULT_SUMMARY_TABLE_COLUMN_MAPPING = {
   [DEFAULT_SUMMARY_TABLE_COLUMNS.UC_ACTIONS]: null,
 };
 
-export const getSummaryTableColumnProperties = (intl, customProps = {}) => {
+const DEFAULT_FORMATTER = (currency) => ({
+  cost: rowData => formatValue(rowData.cost, (value) => formatCost(currency, value)),
```

Comment:
Actually, I mean to move this construction to utilities
```
formatValue(rowData.costPerUse, (value) => formatCost(currency, value))
```

Also, I think that this could be minimized, for example: if we got an callback as a second parameter in `formatValue` we can use here chaining
```suggestion
  cost: rowData => formatValue(rowData.cost, formatCost(currency)),
```
and rewrite `formatCost` to 
```
 const formatCost => currency => value => { ... }
```
What do you mean about it?

Reply:
Let's say we have a function
```
const getCostPerUseCellFormatter = (cellValue, currency) => formatValue(cellValue, (value) => formatCost(currency, value));
```
This would work for `cost` and `costPerUse` columns, but not for `usage` column because it's doesn't represent money. In this case `currency` parameter wouldn't make sense.

I think that trying to generalize this too much creates a lot of abstraction and it's better to have some code duplication than very complex code.